bin/node/runtime/pangolin/src/bridger/s2s/mod.rs

```diff
@@ -0,0 +1,10 @@
+
+
+mod bridger;
```

Comment:
Use bridge instead of bridger,

'bridger' is the rust client implemenation of darwinia bridge relayers, could cause confusion here.